be/src/vec/functions/array/function_array_index.h

```diff
@@ -217,7 +217,7 @@ class FunctionArrayIndex : public IFunction {
         const auto& right_chars = reinterpret_cast<const ColumnString&>(right_column).get_chars();
 
         // prepare return data
-        auto dst = ColumnVector<ResultType>::create(offsets.size());
+        auto dst = ColumnVector<ResultType>::create(offsets.size(), 0);
```

Comment:
Why filled with 0 with dst column, not dst_null_column?

Reply:
dst can be  nullable(uint8) which is nullable(true/false)